src/utils/sanitizeFileName.ts

```diff
@@ -2,12 +2,30 @@
 // eslint-disable-next-line no-control-regex
 const INVALID_CHAR_REGEX = /[\u0000-\u001F"#$&*+,:;<=>?[\]^`{|}\u007F]/g;
 const DRIVE_LETTER_REGEX = /^[a-z]:/i;
+const PERCENT_REGEX = /%/g;
+
+const decodeURIComponentSafe =
+	typeof decodeURIComponent === 'undefined' ? undefined : decodeURIComponent;
```

Comment:
Slight thought, `decodeURIComponent` has been widely supported. I think we can always assume it exists. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent#browser_compatibility

Reply:
Yes. All browsers and node / deno / bun are supported `decodeURIComponent`.
So I also think that we can directly use `decodeURIComponent`.
My initial implementation is super conservative.

Reply:
updated: https://github.com/rollup/rollup/pull/5535/commits/38113286fb17fbe9300efa6609019c9fc0d0c151